modules/kubernetes/pages/k8s-operator/backup-and-restore.adoc

```diff
@@ -0,0 +1,710 @@
+----
+
+The Events section tells you whether the scheduled job was successful.
+
+== Pause or resume a backup schedule
+
+To pause a running backup schedule:
+
+[source.wrap, console]
+----
+kubectl tg backup-schedule pause --cluster-name test-cluster -n tigergraph \
+--tag localdaily
+----
+
+The next backup job will not be scheduled until the schedule is resumed.
+
+To resume a paused schedule:
+
+[source.wrap, console]
+----
+kubectl tg backup-schedule resume --cluster-name test-cluster -n tigergraph \
+--tag localdaily
+----
+
+
+== Backup controls
+
+There are three options to support different implementations of backups:
+
+  --max-retry :                 set max times of retry for each backup
+  --max-backup-file :           set the max number of files you want to retain
+  --max-reserved-day :          set the max number of days you want to retain these backups
+
+You can use them to control the backup job and manage backup files.
+
+=== Maximum number of backup files
+
+Over time, backup files will accumulate and take up disk space.
+Set `--max-backup-file` and `--max-reserved-day`, and the `TigerGraphBackupSchedule` process will help you delete outdated backups automatically based on the strategy you set.
+
+Assume your backup schedule has `--tag daily`.
+If you set `--max-backup-file` to `10`, when a scheduled backup process is completed, a cleaning process runs to remove outdated backups also tagged `daily`.
+Ten backups tagged `daily` are retained, while backups not tagged `daily` will not be affected.
+
+If  you set `--max-reserved-day` to `7`, backups tagged `daily` that were created more than seven days ago will be removed.
+
+[NOTE]
+If the backup process takes too much time, lasting longer than the interval between backups, any missed backups will not be created.
+For example, assume your backup schedule is `0 * * * *`, which creates a backup job once per hour at minute 0.
+If your backup takes 1.5 hours, a backup job will start at 00:00 and end at 01:30, missing the scheduled job at 01:00.
+
+
+== Manage existing backups
+
+=== Delete a backup job
+You can use the following command to delete a backup job.
+
+[source.wrap, console]
+----
+kubectl tg backup delete --cluster-name test-cluster --tag test --namespace tigergraph
+----
+
+=== List backups
+
+[source.wrap, console]
+----
+kubectl tg backup list [OPTIONS]
+
+Options:
+--cluster-name :  (required)set name of the target cluster
+-n, --namespace : set namespace of target cluster
+--tag :           specify the tag of backup
+--json :          output in json format
+--meta :          get the metadata of backup
+----
+
+Run the following commands to list all backups of test-cluster:
+
+[source.wrap, console]
+----
+kubectl tg backup list --cluster-name test-cluster -n tigergraph
+----
+Append the `--json` flag to return the list in JSON format.
+
+If you want to perform a cross-cluster restore, you should get metadata of a backup:
```

Comment:
Are we pointing out a necessary step? If so, I think we need to explain better **when** I need to get this metadata and **what I will need to do with it**. Otherwise, the user still might not do enough or do it correctly.